beacon_node/lighthouse_network/src/rpc/config.rs

```diff
@@ -100,24 +100,30 @@ pub struct RateLimiterConfig {
 }
 
 impl RateLimiterConfig {
-    pub const DEFAULT_PING_QUOTA: Quota = Quota::n_every(2, 10);
-    pub const DEFAULT_META_DATA_QUOTA: Quota = Quota::n_every(2, 5);
-    pub const DEFAULT_STATUS_QUOTA: Quota = Quota::n_every(5, 15);
+    pub const DEFAULT_PING_QUOTA: Quota = Quota::n_every(NonZeroU64::new(2).unwrap(), 10);
```

Comment:
We could use a `const fn` like the below to ensure compile-time guarantee that the value provided is valid, whereas `unwrap` is performed at runtime: 
https://github.com/sigp/lighthouse/blob/b0c374c1cabd7fd960b4dde9ec9b0b26a960ca0c/consensus/types/src/non_zero_usize.rs#L3-L8

Reply:
Oops I got it wrong here, `unwrap` used on a constant value is evaluated at compile time, please ignore me!